.github/workflows/release.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: |
           echo "CURRENT_TAG=$CURRENT_TAG"
       - name: Setup pack
-        uses: buildpacks/github-actions/setup-pack@29954e5d009d40d4d043993f6f39b8ce3e8d9fa3 # v5.8.7
+        uses: buildpacks/github-actions/setup-pack@8748b2e7ad6dc68c285b5771a0a5fbfeddc3d9dd # v5.8.6
```

Comment:
Don't know how this downgrade happened

Reply:
just resolved it manually, it's possible - I guess - when 2 preview branches are rendered at the same time, casued by 2 parallel changes